src/config_generator.py

```diff
@@ -49,7 +56,6 @@ def init_input(config_dict):
 
 def start_config_generation(config_dict: dict = dict()):
     """ entry point for config generation """
-    run_gui = True
     try:
         gui_window = tk.Tk()
     except tk.TclError:
```

Comment:
should not work if tk is missing, maybe catch a more base exeption?

Reply:
Oh, right, didn't think of that.